dcf/models.py

```diff
@@ -131,3 +131,18 @@ def save(self, *args, **kwargs):
 class Image(models.Model):
     item = models.ForeignKey(Item)
     file = ImageField(_('image'), upload_to='images')
+
+
+@python_2_unicode_compatible
+class Complaint(models.Model):
+    item = models.ForeignKey(Item, verbose_name=_('item'))
+    user = models.ForeignKey(settings.AUTH_USER_MODEL)
+    text = models.TextField(_('text'))
+    date_added = models.DateTimeField(_('date_added'), auto_now_add=True)
```

Comment:
Also can you use `created_at` and `updated_at` field names.
And please create Mixin for it, so we can use same structure for other models as well.
Thanks!